### PR TITLE
Function names to match with routes

### PR DESCRIPTION
**Pull request by arshadkazmi42**
Fixes [#Nexmo/nexmo-developer](https://github.com/Nexmo/nexmo-developer/issues/1000)
------------

Update function names to match  the routes

### CODE REVIEW

**Comment by mheap** [comment]
@tbedford Can you review + pull the updates in to NDP?

File: sms/dlr-flask.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 app = Flask(__name__)
 
 @app.route('/webhooks/delivery-receipt', methods=['GET', 'POST'])
-def delivery_receipt():
+def inbound_sms():
```
Comment (tbedford) [inline sms/dlr-flask.py]: Not correct. This is the delivery receipt webhook, why are you changing it to inbound sms?

File: voice/handle-user-input.py [hunk 2]
```diff
@@ -26,7 +26,7 @@ def answer_call():
 
 
 @app.route("/webhooks/dtmf", methods=['POST'])
-def inout_webhook():
+def input_webhook():
```
Comment (tbedford) [inline voice/handle-user-input.py]: Didn't need fixing, but if we are going to change it to something we should probably change the function name to `dtmf`

**Comment by tbedford** [comment]
If you look at the original issue:

https://github.com/Nexmo/nexmo-developer/issues/1000

You can see that for the inbound sms webhook the associated function was delivery_receipt(). Delivery receipt function name needed to be changed to inbound_sms() so that it matched its associated webhook route.

File: sms/dlr-flask.py (hunk 1 above)
Comment (arshadkazmi42) [reply sms/dlr-flask.py]: Just checked. I did the change other way around. Making this fix now.

File: voice/handle-user-input.py (hunk 2 above)
Comment (arshadkazmi42) [reply voice/handle-user-input.py]: I did this change based on the other language sample code `NodeJs`, 'Java'.
I felt inout_webhook was a type.

Does this change even require?

**Comment by tbedford** [comment]
Thanks!

**Pull request merged**